### ISSUE TITLE
Fails to resolve './components/counter'

### ISSUE THREAD

**Issue by robertwyde** [tomarrell/vim-npr#4]
I have a react app with a /src folder and /src/components folder.
In my /src/index.js I have the following line:
import Counter from './components/counter'
(importing the jsx file /src/components/counter.jsx)
However, vim-npr is failing to resolve the file in the path. I have tried with and without the "resolve" key in my package.json. I am using the latest version.
![2019-07-11-094400_956x526_scrot](https://user-images.githubusercontent.com/15756498/61060694-9166ea00-a3c0-11e9-89a4-9430c18f6117.png)

**Comment by tomarrell** [comment]
Thanks for filing an issue. I've been able to reproduce it with your setup here. I think it has something to do with the fact that you're attempting to resolve a `.jsx` file from a `.js` file, which technically should be possible, so I will look into a solving that and push a fix soon.

**Comment by tomarrell** [comment]
Ah ok, I've found out a solution, sorry just had to get my mind back into vimscript mode for a bit there, been a while since I've touched it.

In the script this variable is defined:
```
let g:vim_npr_file_names = ["", ".js", "/index.js", "/index.jsx", "/src/index.js", "/lib/index.js"]
```
This basically determines the paths that will be appended to whatever your cursor is over when you hit `gf`.

The solution here is to include ".jsx" here, as currently it is only by default looking for ".js" extensions. I've pushed a fix which includes this as a default. You should be able to run `:PlugUpdate` to get the latest version, which should fix your issue. Let me know how that goes for you.

**Comment by tomarrell** [comment]
Solved in b44b6b74be3aeb57da06557340aba27f420aa7bf

**Comment by robertwyde** [comment]
Thanks!!! Have a star!
